project-base/app/config/images.yaml

```diff
@@ -51,3 +51,6 @@
     class: Shopsys\FrameworkBundle\Model\Seo\Page\SeoPage
     types:
         -   name: og
+
+-   name: parameterValue
```

Comment:
How this was not necessary before and is now?

Reply:
Before was this functionality done different way, I have removed custom code and replaced it with generic one.

Reply:
https://github.com/shopsys/shopsys/blob/409a4702d90666344b0350c2d0cdf464aec5337b/project-base/app/src/Model/Product/Parameter/ParameterFacade.php#L137

vs

https://github.com/shopsys/shopsys/blob/c2685e96d25523e57b0db5a037d44ce73e3b5772/packages/framework/src/Model/Product/Parameter/ParameterFacade.php#L217